pkg/cli/scan.go

```diff
@@ -60,6 +60,9 @@ var (
 	// UseColors indicates whether to use color output
 	UseColors bool
 	useColors string // used for flag processing
+
+	// Verbose indicates whether to display all fields in default human readlbe output
+	Verbose bool
```

Comment:
usually `Verbose` or `-v` refers to verbosity in log messages. Would it make sense to change this field name to something else?

Reply:
I had kept it --show-details. Changed it to verbose after discussions with Amir.

Reply:
Per Amir, verbose is ok.